### DISCUSSION TITLE
How to download NordVPN from "go.onelink.me" ?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#2374] [General]
Basically Bromite blocks "go.onelink.me" , but NordVPN uses it for its APK links - https://nordvpn.com/download/android/ (clicking "APK version" opens go.online.me but gets blocked).

How do I go around this issue, assuming I want to use APK versions from website and not ones from Google Play Store.

**Comment by rambhau** [comment]
It gets blocked by Ad-blocker/Filterlist, Try again after turning it off. Apk link is redirected to following url - 

https://downloads.nordcdn.com/apps/android/generic/nordvpn-sideload/latest/NordVPN.apk
